server.go

```diff
@@ -169,29 +194,46 @@ func (d *DAServer) HandlePut(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var comm []byte
-	if comm, err = d.store.Put(r.Context(), input); err != nil {
-		d.log.Error("Failed to store commitment to the DA server", "err", err, "comm", comm)
-		w.WriteHeader(http.StatusInternalServerError)
+	if comm, err = svr.store.Put(r.Context(), input); err != nil {
+		svr.WriteInternalError(w, err)
 		return
 	}
 
 	// write out encoded commitment
-	if _, err := w.Write(eigenda.Commitment.Encode(comm)); err != nil {
-		w.WriteHeader(http.StatusInternalServerError)
-		return
+	svr.WriteResponse(w, comm)
```

Comment:
we should still be encoding the commitment

Reply:
good catch